packages/fx-core/src/component/question.ts

```diff
@@ -216,7 +216,14 @@ export async function getQuestionsForDeployV3(
       title: getLocalizedString("core.aad.includeAadQuestionTitle"),
       default: "no",
     });
+    const aadManifestFilePathNode = new QTreeNode({
+      name: Constants.AAD_MANIFEST_FILE,
+      type: "singleFile",
+      title: getLocalizedString("core.aad.aadManifestFilePath"),
+      default: "",
+    });
     node.addChild(aadNode);
+    node.addChild(aadManifestFilePathNode);
```

Comment:
May need to skip this question for vsc?
You can check condition key for QtreeNode to support this. For detail you can reach out for huajie

Reply:
thanks, code updated.